### DISCUSSION TITLE
2022-04-13 Weekly Meeting 51 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#214] [Weekly Meetings]
Weekly Meeting 51 is happening on Wednesday April 13th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 
[OMI Elections](https://github.com/omigroup/omigroup/discussions/211)

Working Group Updates:

- OMI Website update
- OMI-Chan group
- Scripting group
- glTF Extensions Group
- https://github.com/omigroup/media
- any others?

**Comment by madjin** [comment]
Currently doing a lot via M3 avatar interop group: https://github.com/M3-org/avatar-interop
We just had a demo night last week, plus weekly presentations / workshops: https://twitter.com/dankvr/status/1512521045579902976

M3 could act as a partner to help facilitate events together with OMI
I think that's sort of a different relationship than OMI leadership team tho
Thoughts?

**Comment by mrmetaverse** [comment]
New content / media team kicked off yesterday

**Reply by mrmetaverse** [reply]
https://github.com/omigroup/media

**Comment by mrmetaverse** [comment]
Reminder: we have a monthly retrospective to reflect, and turn potential gaps into opportunities and action items :)

**Comment by mrmetaverse** [comment]
We also have two brand new holistic backlogs!! :fire: 
- OMI group holistic backlog: https://github.com/orgs/omigroup/projects/7
- OMI group holistic content backlog: https://github.com/orgs/omigroup/projects/6

**Reply by mrmetaverse** [reply]
the first one is using the new github beta projects board (which they kinda force on us as the new default for backlogs anyways). This allows us to track all tickets across all working groups both in kanban form AND list view. 

The second allow us to track potential content related items (like we sometimes see in website working group aka blog).

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/omigroup/pull/215/files 

Need a vote to formally adopt these minimum requirements for forming a working group. This includes a link to the default templates and steps.

**Comment by mrmetaverse** [comment]
![Screenshot_20220413-130538.png](https://user-images.githubusercontent.com/63426722/163233243-19596d05-c7f1-4911-9e21-9bf3eeeae24e.png)
